dataeng/jobs/createJobsNew.groovy

```diff
@@ -31,7 +32,7 @@ def taskMap = [
     EMR_COST_REPORTER_JOB: EmrCostReporterJob,
     EXPIRE_VERTICA_PASSWORD_JOB: ExpireVerticaPasswordJob,
     SNOWFLAKE_EXPIRE_PASSWORDS_JOB: SnowflakeExpirePasswordsJob,
-    SNOWFLAKE_QUEUE_DEPTH_JOB: SnowflakeQueueDepthJob,
+    SNOWFLAKE_COLLECT_METRICS_JOB: SnowflakeQueueDepthJob, SnowflakeCreditUsageJob,
```

Comment:
Actually, how do you know this works?  does this map a key to a list?  I would actually expect this to break the syntax of the map.  Even if it does create a list value, see this later line:

```
task.value(this, commonConfigMap + extraVars)
```

value is treated as a callable, but this wouldn't work if value is a list.

Reply:
I see, hmm... I was trying to condense the repetitive code by setting up a way to have these (and future) metric collection scripts point to the same file in analytics-secure. I could still try to introduce some if...else logic to the `for (task in taskMap)` loop like: if the data type is a list, iterate through the values in the list.

Or I could just do it the usual way and not rock the boat too much...

Reply:
Basically this: 
`if (task.value instanceof List) {
        task.value.forEach(this, commonConfigMap + extraVars)
    } else {
        task.value(this, commonConfigMap + extraVars)
    }`